frontend/e2e/protractor.conf.js

```diff
@@ -0,0 +1,31 @@
+// Protractor configuration file, see link for more information
+// https://github.com/angular/protractor/blob/master/lib/config.ts
+
+const { SpecReporter } = require('jasmine-spec-reporter');
+
+exports.config = {
+  allScriptsTimeout: 11000,
+  specs: [
+    './**/*.e2e-spec.ts'
```

Comment:
In order to get the tests to run properly on my machine I had to change this line to 
``    './src/*.e2e-spec.ts' ``
Otherwise it was trying to run any test it could find in my node modules folder